src/audio/asrc/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+# SPDX-License-Identifier: BSD-3-Clause
+
+add_local_sources(sof asrc.c IasSrcFarrow.c IasSrcFarrow_generic.c IasSrcFarrow_hifi3.c)
```

Comment:
Do we really need CamelCase C source files? It's inconsistent

Reply:
Yep, I'd like to get rid of the style. I didn't edit it manually due to worry about mistakes but maybe some regexp script could do it. I'll check if such works (camelCase -> snake_case).

Reply:
Oh you meant only file names, that's an easy fix that I will definitely do. But like said I'd like to rename also all the functions, structs, variables.

Reply:
I was writing about file names, but symbols in code also shouldn't be CamelCase ;)

Reply:
I should have all of the typedefs removed now but I'll re-check. Fortunately I can run the code in both device and testbench so the names change should be quite safe.